types.go

```diff
@@ -85,6 +86,17 @@ func NewEntityUID(typ EntityType, id String) EntityUID {
 	return types.NewEntityUID(typ, id)
 }
 
+// NewEntityUIDSet returns an EntityUIDSet ready for use. Optionally, a desired size for the set can be passed as an
```

Comment:
Not sure how useful this function is.

Reply:
Seems it is really only used in tests.  I'd just stick with the second, make it variadic.  Get rid of `FromSlice`